helix-term/src/commands.rs

```diff
@@ -5826,9 +5838,12 @@ fn shell_prompt(cx: &mut Context, prompt: Cow<'static, str>, behavior: ShellBeha
     );
 }
 
-fn suspend(_cx: &mut Context) {
+fn suspend(cx: &mut Context) {
     #[cfg(not(windows))]
-    signal_hook::low_level::raise(signal_hook::consts::signal::SIGTSTP).unwrap();
+    {
+        cx.block_try_flush_writes();
```

Comment:
Why is this not called on windows too ? The hook is unix only yes, but this method is not right ?

Reply:
C-z is supposed to be a noop on windows